### DISCUSSION TITLE
Synthetic seismic + fault generator?

### DISCUSSION THREAD

**Discussion by lindsaymcewan** [softwareunderground/transform-2021-hackathon#20] [Projects]
Anyone interested in trying to create a synthetic seismic + fault generator? Would be useful for training ML models. I think.

**Comment by RichardScottOZ** [comment]
Maybe - I am interested in building fault models, anyway. e.g like from the below fork, so not just seismic.

https://github.com/RichardScottOZ/LineamentLearning

As you saw though, being able to make different features etc. could be handy!
